### PR TITLE
[W5][W14-1]Chang Lei

### PR DESCRIPTION
**Pull request by lallanachang**
Change the find command to the case-insensitive search in AB2, with user guide, JUnit tests and I/O tests updated.

### CODE REVIEW

**Comment by jeraldtsy** [comment]
1) Consider incrementally coding and having more commits 
2) Good attempt at refactoring and removing unnecessary lines

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (kylase-learning) [inline src/seedu/addressbook/commands/FindCommand.java:3]: According to the [Google Style Guide which we are using for this course](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports), wildcard imports is not used.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -50,8 +46,14 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+            for (Iterator iterator = keywords.iterator(); iterator.hasNext();){
```
Comment (kylase-learning) [inline src/seedu/addressbook/commands/FindCommand.java:49]: There is an [API ](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html#equalsIgnoreCase-java.lang.String-)in Java `String` that can help you to case-insensitive comparison.

**Review by kylase-learning — COMMENTED** [review]
Great to see you have submitted your changes via your own branch and done the documentation.

Try to make your commit message more meaningful.

Please close this PR after reading comments.

File: src/seedu/addressbook/commands/FindCommand.java (hunk 2 above)
Comment (lallanachang) [reply src/seedu/addressbook/commands/FindCommand.java:49]: thanks!

**Pull request closed without merging**